# XXX <linux/kcmp.h> since Linux 3.5 + missing hdr for aarch64
#     test_mkfds: SIOCGSKNS since Linux 4.10

--- a/misc-utils/meson.build
+++ b/misc-utils/meson.build
@@ -39,23 +39,23 @@ lsblk_sources = files(
   'lsblk.h',
 )
 
-lsfd_sources = files (
-  'lsfd.c',
-  'lsfd.h',
-  'lsfd-filter.h',
-  'lsfd-filter.c',
-  'lsfd-counter.h',
-  'lsfd-counter.c',
-  'lsfd-decode-file-flags.c',
-  'lsfd-file.c',
-  'lsfd-cdev.c',
-  'lsfd-bdev.c',
-  'lsfd-sock.c',
-  'lsfd-sock.h',
-  'lsfd-sock-xinfo.c',
-  'lsfd-unkn.c',
-  'lsfd-fifo.c',
-)
+#lsfd_sources = files (
+#  'lsfd.c',
+#  'lsfd.h',
+#  'lsfd-filter.h',
+#  'lsfd-filter.c',
+#  'lsfd-counter.h',
+#  'lsfd-counter.c',
+#  'lsfd-decode-file-flags.c',
+#  'lsfd-file.c',
+#  'lsfd-cdev.c',
+#  'lsfd-bdev.c',
+#  'lsfd-sock.c',
+#  'lsfd-sock.h',
+#  'lsfd-sock-xinfo.c',
+#  'lsfd-unkn.c',
+#  'lsfd-fifo.c',
+#)
 
 uuidgen_sources = files(
   'uuidgen.c',
--- a/meson.build
+++ b/meson.build
@@ -2586,18 +2586,18 @@ if not is_disabler(exe)
   bashcompletions += ['lsblk']
 endif
 
-exe = executable(
-  'lsfd',
-  lsfd_sources,
-  include_directories : includes,
-  link_with : [lib_common,
-               lib_smartcols],
-  install_dir : usrbin_exec_dir,
-  install : true)
-if not is_disabler(exe)
-  exes += exe
-  manadocs += ['misc-utils/lsfd.1.adoc']
-endif
+#exe = executable(
+#  'lsfd',
+#  lsfd_sources,
+#  include_directories : includes,
+#  link_with : [lib_common,
+#               lib_smartcols],
+#  install_dir : usrbin_exec_dir,
+#  install : true)
+#if not is_disabler(exe)
+#  exes += exe
+#  manadocs += ['misc-utils/lsfd.1.adoc']
+#endif
 
 exe = executable(
   'uuidgen',
@@ -3275,13 +3275,13 @@ exe = executable(
   include_directories : includes)
 exes += exe
 
-if LINUX
-  exe = executable(
-    'test_mkfds',
-    'tests/helpers/test_mkfds.c',
-    include_directories : includes)
-  exes += exe
-endif
+#if LINUX
+#  exe = executable(
+#    'test_mkfds',
+#    'tests/helpers/test_mkfds.c',
+#    include_directories : includes)
+#  exes += exe
+#endif
 
 exe = executable(
   'test_enosys',
